### DISCUSSION TITLE
Itineraire calculation doesn't work. datafile E0_N45.rd5 not found

### DISCUSSION THREAD

**Discussion by Djey51** [osmandapp/OsmAnd#14641] [General]
Hello,
Could you help me please?

### Description
Impossible to calculate a car itineraire.
I always obtain the following message:
"Impossible de calculer l'itineraire: datafile E0_N45.rd5 not found"

It works well for biking or walking itineraire

### Your Environment
OsmAnd Version: 4.1.11
Android/iOS version: Android 12L
Device model: Xiaomi mi 9T pro

**Maps used (online or offline):**   
If you have an issue related to offline maps, tell us the exact name of the map file where the issue occurs and its edition date.

**Comment by vshcherb** [comment]
This is likely related to B-Router, OsmAnd has other file formats

**Comment by Djey51** [comment]
And do you know how avoid to face this issue?

**Comment by Djey51** [comment]
I modified some settings in BRouter and now it is OK.
Selected car maps from server and add BRouter as exception in battery optimization.
Also download some map in BRouter. I don't know exactly which parameter solved my issue.
Thanks to put me on the good way with BRouter.
Best regards
